spec/integration/post/index_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :system do
+  before(:all) do
+    @user = User.first
+
+    post1 = Post.create(author: @user, title: 'Hello1', text: 'This is my first post')
+    post2 = Post.create(author: @user, title: 'Hello2', text: 'This is my first post')
```

Comment:
- [ ] All your tests are failing. Most of the test errors indicate the tests are being carried out on non-existent records. Kindly note that in test files/environments the data needed to run the tests should be readily available. For example, here you are calling `User.first` but there is no scenario in which you create a User in this file. There is also no created user in your seed file so these tests should fail.

  Kindly ensure you create the users as seed data or as part of the test files and ensure the tests are passing.

  ![image](https://user-images.githubusercontent.com/36542986/232798522-8fc33dc6-0eca-48ba-8bb4-e8ebacd5725d.png)